library/src/blas2/rocblas_ger.cpp

```diff
@@ -9,14 +9,37 @@
 
 namespace
 {
-    template <typename>
+    template <bool, typename>
     constexpr char rocblas_ger_name[] = "unknown";
     template <>
-    constexpr char rocblas_ger_name<float>[] = "rocblas_sger";
+    constexpr char rocblas_ger_name<false, float>[] = "rocblas_sger";
```

Comment:
reversed order ? sometimes it is <bool,typename> and sometimes <typename, bool>.
would worth it to make the code uniform.

Reply:
You need to use `rocblas_operation` instead of boolean. When comparing against transpose, you compare against `rocblas_operation_none`, i.e., use:
```
if(trans != rocblas_operation_none)
```
instead of:
```
if(trans == rocblas_operation_transpose)
```
In real functions, `rocblas_operation_conjugate_transpose` should be considered identical to `rocblas_operation_transpose`.

Reply:
I followed the two existing patterns, in kernels the code modifiers came first, while in testing they came second (see dot).    I think it may have been for default value use was what led to the latter, but I don't have that need as it can be explicitly set.   I am fine with either or both, someone will complain either way as it doesn't match elsewhere in the framework.  Thus I chose to match each framework's implicit convention.

Reply:
Lee this bool only controls conjugation not any transpose operation, see dot for the pattern bool CONJ, or Level 3 bool BATCHED variants.